Android/app/src/main/java/com/example/samplestickerapp/EntryActivity.java

```diff
@@ -12,7 +12,7 @@
 import android.content.Intent;
 import android.os.AsyncTask;
 import android.os.Bundle;
-import android.support.annotation.Nullable;
+import androidx.annotation.Nullable;
```

Comment:
please do a reformat, and that will move this line to the bottom.

Reply:
Code reformatted!